Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Themes/InteractableThemeShaderUtils.cs

```diff
@@ -107,18 +109,7 @@ public static Material GetValidMaterial(Renderer renderer)
 
             if (renderer != null)
             {
-#if UNITY_EDITOR
-                if (!Application.isPlaying)
-                {
-                    material = renderer.sharedMaterial;
-                }
-                else
-                {
-                    material = renderer.material;
-                }
-#else
-                material = renderer.material;
-#endif
+                material = renderer.sharedMaterial;
```

Comment:
My guess is that this was written the original way because, in the editor while playing, making changes to the `sharedMaterial` propagates those changes back the the material file. Not sure we should update by just removing the original logic.

Reply:
It's also the Unity recommendation to [not modify materials returned by sharedMaterial](https://docs.unity3d.com/ScriptReference/Renderer-sharedMaterial.html), but as long as the caller of this method doesn't actually modify the material, we might be fine.

Reply:
Oh though I misread the original version, which didn't use `sharedMaterial` outside the editor.

Reply:
Yep! All callers of this method only read from the material properties.